crates/apub/src/activities/community/block_user.rs

```diff
@@ -101,6 +101,8 @@ impl ActivityHandler for BlockUserFromCommunity {
     let community_user_ban_form = CommunityPersonBanForm {
       community_id: community.id,
       person_id: blocked_user.id,
+      // TODO how to carry across the ban expiration time?
+      expires: None,
```

Comment:
The only thing I'm not sure how to do for this PR.

Reply:
You mean how to federate it? I dont know if or how other projects do this, so i would simply add a field like `expires` with timestamp. And add it to the test file as well for documentation.

Reply:
mmk, I'll see if I can figure out how to do that.

Reply:
Sorry I wasnt really clear where to make the changes. For community bans, you should add the `expires` field in `protocol::activities::community::block_user`, then you can use it in this file to send/receive them. Also add the field to `assets/lemmy/activities/community/block_user.json` to make sure that it parses properly (and maybe in api_tests as well).